Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/uguirenderer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class uguirenderer : MonoBehaviour
```

Comment:
Is this class meant to be here? Not exactly sure what it's purpose is.
If it is on purpose, can you update
spacing, comments, remove empty update method, name of class casing is off, and no MSFT copyright, etc

Reply:
no. this was testing something. Not meant to go in. Good catch. I will remove.